Library/Homebrew/test/test_dependency.rb

```diff
@@ -48,7 +48,7 @@ def test_merge_repeats
     assert_equal Dependency, merged.first.class
 
     foo_named_dep = merged.find {|d| d.name == "foo"}
-    assert_equal [:build, "bar"], foo_named_dep.tags
+    assert_equal ["bar"], foo_named_dep.tags
```

Comment:
Nope, I agree with the overall changes but this seems to be a mistake/regression.